server/webhook.go

```diff
@@ -836,9 +867,10 @@ func (s *Server) createIssueFromPushEvent(ctx context.Context, pushEvent *vcs.Pu
 	if migrationType == db.Data {
 		issueType = api.IssueDatabaseDataUpdate
 	}
+	commitsMsg := getCommitsMessageShort(pushEvent.CommitList)
 	issueCreate := &api.IssueCreate{
 		ProjectID:     repo.ProjectID,
-		Name:          fmt.Sprintf("%s by %s", migrationDescription, strings.TrimPrefix(fileEscaped, repo.BaseDirectory+"/")),
+		Name:          fmt.Sprintf("%s by %s", migrationType, commitsMsg),
```

Comment:
This name will not be readable and not helpful, especially since we allow users to edit the file with a later commit.